src/Compilers/VisualBasic/Test/IOperation/IOperation/IOperationTests_IBinaryOperatorExpression.vb

```diff
@@ -7058,5 +7058,70 @@ Block[B5] - Exit
             VerifyFlowGraphAndDiagnosticsForTest(Of MethodBlockSyntax)(source, expectedGraph, expectedDiagnostics)
         End Sub
 
+        <CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)>
+        <Fact()>
+        Public Sub LogicalFlow_01()
+            Dim source = <![CDATA[
+Public Class Class1
+    Sub M(a As Boolean, b As Boolean, c As Boolean)'BIND:"Sub M(a As Boolean, b As Boolean, c As Boolean)"
+        a = b AndAlso (Not c)
```

Comment:
>a = b AndAlso (Not c) [](start = 8, length = 21)

This scenario looks sufficiently good to demonstrate the reuse of existing Not operator, but I think it doesn't quite show the impact from skipping parenthesized.
Try adding tests for ```a = b OrElse (Not c andalso Not d)``` and ```a = b OrElse Not (c orElse d)```. The should produce graphs equivalent to ```a = b OrElse Not c andalso Not d```, modulo the syntax associated with nodes.